bridge/MixerManager.cpp

```diff
@@ -82,18 +82,26 @@ Mixer* MixerManager::create(uint32_t lastN)
     const auto localVideoSsrc = _ssrcGenerator.next();
 
     std::vector<uint32_t> audioSsrcs;
-    std::vector<SimulcastLevel> videoSsrcs;
+    std::vector<SimulcastGroup> videoSsrcs;
     std::vector<SimulcastLevel> videoPinSsrcs;
 
     for (uint32_t i = 0; i < _config.audio.lastN + _config.audio.lastNextra; ++i)
     {
         audioSsrcs.push_back(_ssrcGenerator.next());
     }
 
-    // Last-n + screen share, extra
-    for (uint32_t i = 0; i < lastN + 3; ++i)
+    // screen share / slides
     {
-        videoSsrcs.push_back({_ssrcGenerator.next(), _ssrcGenerator.next()});
+        SimulcastLevel a[1] = {{_ssrcGenerator.next(), _ssrcGenerator.next(), false}};
```

Comment:
Generate 6 target ssrcs per rewrite target. Before we had only 2 ssrcs as each client only receives one vieo stream from each forwarded participant. That is still true, but we will need to be ready to forward 3 streams per forwarded participant over the barbell.